tests/conftest.py

```
@@ -1362,7 +1365,23 @@ def factory(
+                            erasure_coded=erasure_coded,
+                        )
+                    else:
+                        full_pool_name = helpers.create_cephfs_data_pool(
+                            pool_name=constants.CUSTOM_CEPHFS_POOL,
+                            compression=ocsci_config.ENV_DATA.get("compression")
+                            or compression,
+                            replica=ocsci_config.ENV_DATA.get("replica") or replica,
+                        )
+                    interface_name = full_pool_name
+                else:
+                    if pool_name is None:
```

Ruff (0.15.20)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)